### ISSUE TITLE
Rename JSR package?

### ISSUE THREAD

**Issue by lukeed** [denodrivers/postgres#482]
I initially thought `@bartlomieju/postgres` was a fork – especially since it's so far behind (#480) – because I know this package as a `denodriver/...` package. Is there not a `@deno/` scope or maybe a `@denox/` ?

I understand not wanting to nest this under `@std` but I think the current naming loses clarity/authoritativeness. But maybe that's just me 🤷

**Comment by drewbitt** [comment]
Yes please. I am the same way; I knew it was at least semi-official (because I recognized @bartlomieju) but most people a) wouldn't recognize and b) I was still unsure it was the exact same, and as you mention it's actually behind.

**Comment by sachaw** [comment]
why not use `@denodrivers/postgres`

**Comment by allout58** [comment]
Also to add to the confusion, there's a package that looks like maybe a minimal fork here: https://jsr.io/@dewars/postgres@0.20.0 that has a newer version than is in this repo. Without clarity, the official JSR package looks just the same (maybe even worse since 0.17 < 0.20)

**Comment by av** [comment]
New to Deno, wanted to query Postgres with 2.0, got quite confused between JSR, `@std`, deno.land. It's clear that `deno.land` imports are still working. Some high-level guidance would be appreciated.

**Comment by sachaw** [comment]
> Also to add to the confusion, there's a package that looks like maybe a minimal fork here: https://jsr.io/@dewars/postgres@0.20.0 that has a newer version than is in this repo. Without clarity, the official JSR package looks just the same (maybe even worse since 0.17 < 0.20)

I published that for my own usage, based on my open pr in this repo.

**Comment by sachaw** [comment]
> New to Deno, wanted to query Postgres with 2.0, got quite confused between JSR, `@std`, deno.land. It's clear that `deno.land` imports are still working. Some high-level guidance would be appreciated.

JSR is replacing deno.land, deno.land is avaliable for backwards compat, do not use it going forward.

**Comment by sant123** [comment]
https://deno.land/std and https://deno.land/x were the way to import packages/modules in Deno. However with the launch of JSR everything from `std` was moved to it and new packages/modules from third parties should be created there as well. The goal of this issue is to deploy a stable/supported/updated version of the PostgreSQL driver in JSR, so my suggestion is to use the drivers on NPM meanwhile this is resolved or a third party publish it to JSR. 

[pg](https://www.npmjs.com/package/pg), [postgres](https://www.npmjs.com/package/postgres) and [postgrejs](https://www.npmjs.com/package/postgrejs) are good options.
